tests/inspectdb/tests.py

```diff
@@ -299,3 +299,16 @@ def test_introspection_errors(self):
         self.assertIn("# Unable to inspect table 'nonexistent'", output)
         # The error message depends on the backend
         self.assertIn("# The error was:", output)
+
+    def test_indexes_meta(self):
+        out = StringIO()
+        call_command('inspectdb', table_name_filter=lambda tn: tn.startswith('inspectdb_indexes'), stdout=out)
+        output = out.getvalue()
+        self.assertIn('        indexes = [', output, msg='inspectdb should generate indexes of the models.')
+        # Single column index via db_index and auto-generated name
```

Comment:
Does the introspected field also have `db_index=True`? Is it an error to specify `db_index=True` and also specify a single column index in `Meta.indexes`?